### PR TITLE
35 refactor lambo objective and wrapper

### PR DESCRIPTION
**Pull request by RMichae1**
Added erroneous sequences to module init.
Added LamBO hydra task config to objective_repository/foldx_rfp_lambo as YAML file.
Read LamBO from hydra config on register.
Cleanup WrapperFactory.
Cleanup RFPWrapper.

### CODE REVIEW

File: src/poli/objective_repository/foldx_rfp_lambo/register.py [hunk 1]
```diff
@@ -88,40 +87,28 @@ def create(
...
-            # I've removed the initial SK and some MYG in the middle
-            # Then it's really just a K in position 159 which is different.
-            # seq[159] = 'K'  # strings are immutable
-            seq = "GEELIKENMHMKLYMEGTVNNHHFKCTTEGEGKPYEGTQTQRIKVVEGGPLPFAFDILATCFSKTFINHTQGIPDFFKQSFPEGFTWERVTTYEDGGVLTVTQDTSLQDGCLIYNVKLRGVNFPSNGPVMQKKTLGWEATTETLYPADGGLEGRCDMALKLVGGGHLHCNLKTTYRSKKPAKNLKMPGVYFVDRRLERIKEADNETYVEQHEVAVARYCDLPSKL"
-            all_seqs[np.where(all_seqs == problem_seq)] = seq
+        if self.problem_sequence in all_seqs:
+            # substitute erroneous sequence "X in position 159" with correct PDB fasta sequence
+            all_seqs[
+                np.where(all_seqs == self.problem_sequence)
+            ] = self.correct_sequence
```
Comment (miguelgondu) [inline src/poli/objective_repository/foldx_rfp_lambo/register.py:94]: Much cleaner, I really like it!

File: src/poli/objective_repository/foldx_rfp_lambo/register.py [hunk 2]
```diff
@@ -88,40 +87,28 @@ def create(
...
-# __name__ = '__main__'
-# config_path = os.path.join(project_root, 'hydra_config')
+# satisfy Hydra
 def get_config():
     global conf
+    task = yaml.safe_load(
+        Path(
+            os.path.dirname(__file__) + os.path.sep + "lambo_task_config.yaml"
+        ).read_text()
+    )
```
Comment (miguelgondu) [inline src/poli/objective_repository/foldx_rfp_lambo/register.py:111]: yaml's already on the dependencies of `poli__lambo`, right?

Comment (miguelgondu) [reply src/poli/objective_repository/foldx_rfp_lambo/register.py:111]: not explicitly, but I'm sure it must be a part of `hydra`'s dependencies, right?

Comment (miguelgondu) [reply src/poli/objective_repository/foldx_rfp_lambo/register.py:111]: Yeah, my `poli__lambo` has `PyYAML==6.0.1`. I'll assume it comes implicitly. Looks good to me!

**Review by miguelgondu — APPROVED** [review]

File: src/poli/objective_repository/foldx_rfp_lambo/register.py (hunk 2 above)
Comment (RMichae1) [reply src/poli/objective_repository/foldx_rfp_lambo/register.py:111]: Yes. My `pol__lambo` did have yaml.

**Pull request merged**